src/prefect/environments/storage/docker.py

```diff
@@ -500,9 +520,11 @@ def create_dockerfile_object(self, directory: str) -> str:
                 with open(flow_path, "wb") as f:
                     cloudpickle.dump(self._flows[flow_name], f)
                 copy_flows += "COPY {source} {dest}\n".format(
-                    source=flow_path.replace("\\", "/")
-                    if self.dockerfile
-                    else "{}.flow".format(clean_name),
+                    source=(
+                        flow_path.replace("\\", "/")
+                        if self.dockerfile
+                        else "{}.flow".format(clean_name)
+                    ),
```

Comment:
Readability change, just wrapped in parens so it's clear it's all for the `source` arg